app/Services/ToggleNameToNameList.php

```diff
@@ -23,9 +23,8 @@ public function execute(): void
 
     private function addOrRemove(): void
     {
-        $list = NameList::where('user_id', auth()->id())
-            ->where('id', $this->listId)
-            ->firstOrFail();
+        $list = auth()->user()->lists() 
+            ->findOrFail($this->listId);
```

Comment:
Warning: it must be First or Fail, not find or fail. We need to get the first one.

Reply:
That's exactly the same actually ...